playground/src/index.hbs

```diff
@@ -96,7 +92,9 @@
         <div class="preview-sizer"></div>
         <iframe frameborder="0" id="previewIframe" title="AMP Playground Output" class="fadein" style="width: 375px; height: 667px;"></iframe>
       </div>
+
     </section>
     </main>
+    <div id="error-list" class="fly-in fly-in--right fly-in--error" data-title="Error List"></div>
```

Comment:
While testing it just now I've been wondering wether "Validation errors" might be a more helpful title. It being a list is kinda obvious... WDYT?